presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q36.result

```diff
@@ -1,101 +1,101 @@
--- delimiter: |; types: DECIMAL|LONGNVARCHAR|CHAR|INTEGER|BIGINT
```

Comment:
@kokosing do you remember why this is left over here? Some columns are defined as CHAR but others are still on LONGNVARCHAR?